core/src/main/java/apoc/cypher/Cypher.java

```diff
@@ -81,103 +79,31 @@ public Stream<MapResult> run(@Name("statement") String statement, @Name("params"
         return runCypherQuery(tx, statement, params);
     }
 
-    private Stream<RowResult> runManyStatements(
-            Reader reader,
-            Map<String, Object> params,
-            boolean schemaOperation,
-            boolean addStatistics,
-            int queueCapacity) {
-        BlockingQueue<RowResult> queue = runInSeparateThreadAndSendTombstone(
-                queueCapacity,
-                internalQueue -> {
-                    if (schemaOperation) {
-                        runSchemaStatementsInTx(reader, internalQueue, params, addStatistics);
-                    } else {
-                        runDataStatementsInTx(reader, internalQueue, params, addStatistics);
-                    }
-                },
-                RowResult.TOMBSTONE);
-        return StreamSupport.stream(
-                new QueueBasedSpliterator<>(queue, RowResult.TOMBSTONE, terminationGuard, Integer.MAX_VALUE), false);
-    }
-
-    private <T> BlockingQueue<T> runInSeparateThreadAndSendTombstone(
-            int queueCapacity, Consumer<BlockingQueue<T>> action, T tombstone) {
-        /* NB: this must not be called via an existing thread pool - otherwise we could run into a deadlock
-          other jobs using the same pool might completely exhaust at and the thread sending TOMBSTONE will
-          wait in the pool's job queue.
-        */
-        BlockingQueue<T> queue = new ArrayBlockingQueue<>(queueCapacity);
-        Util.newDaemonThread(() -> {
-                    try {
-                        action.accept(queue);
-                    } finally {
-                        while (true) { // ensure we send TOMBSTONE even if there's an InterruptedException
-                            try {
-                                queue.put(tombstone);
-                                return;
-                            } catch (InterruptedException e) {
-                                Thread.currentThread().interrupt();
-                            }
-                        }
-                    }
-                })
-                .start();
-        return queue;
-    }
-
-    private void runDataStatementsInTx(
-            Reader reader, BlockingQueue<RowResult> queue, Map<String, Object> params, boolean addStatistics) {
-        Scanner scanner = new Scanner(reader);
-        scanner.useDelimiter(";\r?\n");
-        while (scanner.hasNext()) {
-            String stmt = removeShellControlCommands(scanner.next());
-            if (stmt.trim().isEmpty()) continue;
-            if (!isSchemaOperation(stmt)) {
```

Comment:
Here we used to filter out schema operations, they were simply never run. But why?